tests/selinux.bats

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bats
+
+load helpers
+
+@test "selinux test" {
+  if ! which selinuxenabled ; then
+    skip
+   elif ! /usr/sbin/selinuxenabled; then
+     skip
+  fi
+  image=alpine
+  cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json $image)
+  firstlabel=$(buildah --debug=false run $cid cat /proc/1/attr/current)
+  run buildah --debug=false run $cid cat /proc/1/attr/current
+  [ "$status" -eq 0 ]
+  [ "$output" -eq $firstlabel ]
+
+  cid1=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json $image)
+  run buildah --debug=false run $cid1 cat /proc/1/attr/current
+  [ "$status" -eq 0 ]
+  [ "$output" -ne $firstlabel ]
+
+  buildah rm $cid
+  buildah rm $cid1
+}
+
```

Comment:
The test looks mostly alright (don't be afraid to add comments!), but fails when I run it locally, I expect because the "vfs" driver doesn't support mount labels.  The test is skipped in Travis and in Red Hat CI.

Reply:
Not verified: would adding "libselinux-utils" or "/usr/sbin/selinuxenabled" to the list of packages to install in `.papr.sh` help?

Reply:
I will try.

Reply:
Add libselinux-utils